core/templates/dev/head/base.html

```diff
@@ -280,6 +281,16 @@
               {{ warnings_and_loader() }}
             </div>
 
+            <noscript>
+              <div class="oppia-page-cards-container">
+                <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <h2>We Need JavaScript in Your Browser <img src="/images/general/sad_face.png" /> </h2>
```

Comment:
Hm, not sure "We Need JS in Your Browser" is quite right; the user won't care what "we" need.

For the icon, can we use this instead?

    https://design.google.com/icons/#ic_sentiment_dissatisfied

Basically:  `<i class="material-icons">&#xE811;</i>`. We do this elsewhere in the codebase, too; the aim is to use the material design icon font that's already in place, rather than requiring the browser to make an additional image request.

Thanks!